mmrotate/core/anchor/anchor_generator.py

```diff
@@ -43,6 +44,47 @@ def single_level_grid_priors(self,
         return anchors
 
 
+@ROTATED_ANCHOR_GENERATORS.register_module()
+class FakeRotatedAnchorGenerator(AnchorGenerator):
+    """Fake rotate anchor generator for 2D anchor-based detectors."""
+
+    def __init__(self, angle_version='oc', **kwargs):
```

Comment:
AnchorGenetator's parameters must be explicitly passed in the configuration file, otherwise, users may mistakenly adopt the default `oc` horizontal box when using `le90` or `le135`. It's best to give a configuration file to show users how to use it. If it is not used at present, I suggest deleting this part of the code first.